src/lib/gas-refund/multi-staking-utils.ts

```diff
@@ -203,12 +203,18 @@ export function computeAggregatedStakeChainDetails(
       {
         transactionsWithClaimable: transactionsWithClaimableByEpoch[epoch],
 
-        refundedByChain: refundedByEpochByChain[epoch],
-        claimableByChain: claimableByEpochByChain[epoch],
+        refundedByChain: toFixed(refundedByEpochByChain[epoch]),
+        claimableByChain: toFixed(claimableByEpochByChain[epoch]),
       },
     ];
   });
   const byEpoch = Object.fromEntries(entries);
 
   return byEpoch;
 }
+
+
+function toFixed<K  extends string | number | symbol>(dictionary: Record<K, BigNumber>) {
```

Comment:
You need to explicitly assign return type `Record<k,string>`. Ts can't figure it out here and thinks it's `any`